README.md

```diff
@@ -1,16 +1,18 @@
 # Drone plugin for Helm 3
 
-[![Build Status](https://cloud.drone.io/api/badges/pelotech/drone-helm3/status.svg)](https://cloud.drone.io/pelotech/drone-helm3)
-[![Go Report](https://goreportcard.com/badge/github.com/pelotech/drone-helm3)](https://goreportcard.com/report/github.com/pelotech/drone-helm3)
-[![](https://images.microbadger.com/badges/image/pelotech/drone-helm3.svg)](https://microbadger.com/images/pelotech/drone-helm3 "Get your own image badge on microbadger.com")
+[![Build Status](https://drone.corp.mongodb.com/api/badges/mongodb-forks/drone-helm3/status.svg)](https://drone.corp.mongodb.com/mongodb-forks/drone-helm3)
```

Comment:
you should remove this, build status doesn't work outside of our network.

Reply:
/api/badges is now exposed publicly, but I can remove it if required

Reply:
oh, nm